SemanticSearch.md

```diff
@@ -141,91 +140,158 @@ The query response is divided into two parts. First is the parsed_input node, wh
+
+
+2.0 response:
 ```
 {
-	"data": {
-		"extracted_keywords": [{
-			"name": "java",
-			"weight": 0.95,
-			"type": "keyword",
-			"relationships": {}
-		}],
-		"summary": {
-			"job_level": [{
-				"name": "Experienced (non-Manager)",
-				"id": "3",
-				"weight": 1
-			}],
-			"job_titles": [{
-				"name": "Java Developer",
-				"id": "15.2",
-				"weight": 1
-			}],
-			"skills": [{
-				"name": "Java (Programming Language)",
-				"id": "KS120076FGP5WGWYMP0F",
-				"weight": 1
-			}],
-			"occupations": [{
-				"name": "Computer Programmers",
-				"id": "15-1131.00",
-				"weight": 0.98
-			}]
+	"data" : {
+		"parsed_input" : {
+			"input" : "Java developer,Careerbuilder",
+			"parsed_fragments" : [
+				"java developer",
+				"careerbuilder"
+			],
+			"input_to_extracted_keywords" : {
+				"Careerbuilder" : "careerbuilder",
+				"Java developer" : "java developer"
+			}
 		},
-		"versions": {
-			"job_titles": "CaroteneV3",
-			"occupations": "ONet17",
-			"skills": "SkillsV4",
-			"job_level": "JobLevel",
-			"extracted_keywords": "InterestingTermsV3"
+		"extracted_keywords" : [
+			{
+				"name" : "java developer",
+				"relationships" : {
+					"skills" : [
+						{
+							"weight" : 1,
+							"name" : "Java (Programming Language)",
+							"id" : "KS120076FGP5WGWYMP0F"
+						},
+						{
+							"name" : "Hibernate (Java)",
+							"id" : "KS124PR62MV42B5C9S9F",
+							"weight" : 0.93154
+						},
+						{
+							"id" : "KS123KG6DL8N3D5ZW036",
+							"name" : "Java Platform Enterprise Edition (J2EE)",
+							"weight" : 0.92962
+						}
+					],
+					"job_titles" : [
+						{
+							"id" : "15.2",
+							"name" : "Java Developer",
+							"weight" : 1
+						}
+					],
+					"job_level" : [
+						{
+							"name" : "Experienced (non-Manager)",
+							"id" : "3",
+							"weight" : 1
+						}
+					],
+					"text_kernel_related_keywords" : [
+						{
+							"name" : "Java Architect",
+							"weight" : 1
+						},
+						{
+							"name" : "J2EE",
+							"weight" : 1
+						},
+						{
+							"weight" : 1,
+							"name" : "Java Programmer"
+						}
+					],
+					"occupations" : [],
+					"related_keywords" : []
+				},
+				"type" : "job_title",
+				"weight" : 100
+			},
+			{
+				"name" : "careerbuilder",
+				"type" : "company",
+				"weight" : 100
+				"relationships" : {},
+			},
+			{
+				"type" : "job_level",
+				"name" : "EXPERIENCED_NON_MANAGER"
+			},
+			{
+				"type" : "location",
+				"entity_node" : {
+					"address" : "Atlanta, Georgia",
+					"region" : "Georgia"
+				}
+			},
+			{
+				"name" : "Java developer",
+				"type" : "job_title"
+			}
+		],
+		"versions" : {
+			"job_titles" : "CaroteneV3",
+			"text_kernel_related_keywords" : "TextKernelRelatedSearchTerms",
+			"job_level" : "JobLevel",
+			"occupations" : "ONet17",
+			"related_keywords" : "RelatedSearchTermsV1",
+			"extracted_keywords" : "ExtractedKeywords",
+			"skills" : "SkillsV4",
+			"custom_keywords" : "CustomKeywords"
```

Comment:
Note that this response is already outdated, we don't have joblevel anymore.
Besides, we need to do a sumary user know that the extracted_keywords in the response has two kinds of entities, one kind is from solr(name, type, weight, relationship), another kind is from parsed data (from PAN) that has (type and entity_node). currently we have two PAN entities (Location and experience level).